ui/app/AppLayouts/Chat/views/ContactsColumnView.qml

```diff
@@ -294,12 +294,19 @@ Item {
                 }
 
                 onDisplayGroupInfoPopup: {
-                    // Not Refactored Yet
+                    root.chatSectionModule.prepareChatContentModuleForChatId(chatId)
```

Comment:
Please correct me if I'm wrong, but we shouldn't use backend objects directly, that is `prepareChatContentModuleForChatId` should be a part of Stores API.

Reply:
In theory, yes, but this function is being used all over the place, I thought it was ok

Reply:
I am not sure what was the agreement in terms of current refactoring. But I recall @alexandraB99 asked to not use backend objects if possible.